pipeline_dp/dp_engine.py

```diff
@@ -147,8 +150,11 @@ def _bound_contributions(self, col, max_partitions_contributed: int,
         col = self._ops.sample_fixed_per_key(col, max_partitions_contributed,
                                              "Sample per privacy_id")
 
-        # (privacy_id, [(partition_key, aggregator)])
+        self._add_report_stage(
+            f"Contribution bounding: randomly selected not more than "
```

Comment:
Nit: Cross-partition contribution bounding

Reply:
Fixed, thanks for catching!